src/main/java/org/opensearch/security/securityconf/impl/v7/InternalUserV7.java

```diff
@@ -114,9 +128,17 @@ public void setAttributes(Map<String, String> attributes) {
             this.attributes = attributes;
         }
 
+        public boolean getIsEnabled() {
+            return this.isEnabled;
+        }
+
+        public boolean getIsService() {
+            return this.isService;
+        }
+
         @Override
         public String toString() {
-            return "InternalUserV7 [hash=" + hash + ", reserved=" + reserved + ", hidden=" + hidden + ", _static=" + _static + ", backend_roles="
+            return "InternalUserV7 [hash=" + hash + ", isEnabled=" + isEnabled + ", isService=" + isService + ", reserved=" + reserved + ", hidden=" + hidden + ", _static=" + _static + ", backend_roles="
```

Comment:
This is awkward with the mix of casing. Do you think the work `is` is important in these variable names?

Reply:
I would probably keep them, since I prefer the boolean names to say the positive case in them. I think `enabled` would be fine but `service=true/false` sounds ambiguous to me.